### PR TITLE
get quicksight going again with a different template dataset

### PR DESCRIPTION
**Pull request by sastels**
# Summary | Résumé

Hopefully avoid the broken `templates` dataset and also fix a problem with underscores in a few `physical_table_map_id`s.

### CODE REVIEW

File: aws/quicksight/dataset_notification_history.tf [hunk 1]
```diff
@@ -6,7 +6,7 @@ resource "aws_quicksight_data_set" "notification_history" {
   import_mode = "SPICE"
 
   physical_table_map {
-    physical_table_map_id = "notification_history"
```
Comment (sastels) [inline aws/quicksight/dataset_notification_history.tf:9]: underscores not allowed here 😞

File: aws/quicksight/dataset_templates.tf [hunk 2]
```diff
@@ -1,12 +1,12 @@
 # valid column types are [STRING INTEGER DECIMAL DATETIME BIT BOOLEAN JSON]
 
 resource "aws_quicksight_data_set" "templates" {
-  data_set_id = "templates"
```
Comment (sastels) [inline aws/quicksight/dataset_templates.tf:4]: the dataset with id `templates` is horribly broken in AWS and can't be deleted. AWS is looking into it, but for now let's just use a different id

**Review by ben851 — APPROVED** [review]
LGTM!

File: aws/quicksight/dataset_templates.tf (hunk 2 above)
Comment (jimleroyer) [reply aws/quicksight/dataset_templates.tf:4]: That's bad, would we have to nuke it all and redo it to fix it?

**Pull request merged**